packages/tools/locales/pt_BR.po

```diff
@@ -1704,9 +1705,8 @@ msgid "Share notebook..."
 msgstr "Compartilhar caderno..."
 
 #: packages/app-desktop/gui/MainScreen/commands/showShareNoteDialog.js:16
-#, fuzzy
 msgid "Publish note..."
-msgstr "Compartilhar nota..."
+msgstr "Publicar nota…"
```

Comment:
everwhere else `...` is used and not the 1 character ellipsis

Reply:
I'm sorry. I fixed it.

Reply:
No worries. It's just about consistency. Otherwise people might wonder why sometimes one is used instead of the other.